core/src/auctions.ts

```diff
@@ -103,29 +103,30 @@ export const fetchAllInitialAuctions = async function (network: string): Promise
         return fetchAuctionsByCollateralType(network, collateralType);
     });
     const auctionGroups = await Promise.all(auctionGroupsPromises);
-    return auctionGroups.flat();
+    return auctionGroups.flat() as AuctionInitialInfo[];
```

Comment:
Why is it needed?

Reply:
My IDE throws an error here. So I thought just typing the output of `auctionGroups.flat()`, would help fix this issue (which it does). Should I remove it?
![Screenshot 2022-02-22 at 09 47 29](https://user-images.githubusercontent.com/30908158/155095783-866dbef9-e791-4910-8891-2e6e1887b98f.png)

Reply:
Is it only your IDE or our typescript build as well? General rule: type errors doesn't mean you need to fix a type, in most cases it means you're doing something wrong with data structures.